diff -ru pyHS100-0.3.5~/pyHS100.egg-info/requires.txt pyHS100-0.3.5/pyHS100.egg-info/requires.txt
--- pyHS100-0.3.5~/pyHS100.egg-info/requires.txt	2019-04-12 22:15:09.000000000 -0700
+++ pyHS100-0.3.5/pyHS100.egg-info/requires.txt	2020-04-30 15:17:17.505130786 -0700
@@ -1,3 +1,2 @@
 click
 click-datetime
-typing
diff -ru pyHS100-0.3.5~/setup.py pyHS100-0.3.5/setup.py
--- pyHS100-0.3.5~/setup.py	2019-04-12 22:12:57.000000000 -0700
+++ pyHS100-0.3.5/setup.py	2020-04-30 15:17:01.513681291 -0700
@@ -8,7 +8,7 @@
       author_email='sean@gadgetreactor.com',
       license='GPLv3',
       packages=['pyHS100'],
-      install_requires=['click', 'click-datetime', 'typing'],
+      install_requires=['click', 'click-datetime'],
       python_requires='>=3.4',
       entry_points={
             'console_scripts': [
